R/scmake.R

```diff
@@ -252,9 +252,9 @@ is_ind_file <- function(target_names, ind_ext=getOption("scipiper.ind_ext")) {
 #' as_ind_file('mydata.rds', ind_ext='st') # 'mydata.rds.st'
 #' as_ind_file('mydata.rds.ind') # Error: "data_file is an indicator file already"
 as_ind_file <- function(data_file, ind_ext=getOption("scipiper.ind_ext")) {
```

Comment:
is there a convention on whether to pluralize names of function arguments if they are vectors?

Reply:
Hmm, the usually-vectorized nature of R blurs that line a lot. Here are the first few examples I can find with named, vectorized arguments:
* `dir(path, ...)`, where path can be a vector
* `log(x)` and `exp(x)`, where `x` can be a vector
* `ifelse(test, yes, no)` where all 3 args can be vectors

So I think the answer is no, or if there is such a convention, it's not widely followed?